packages/components/src/components/select-combobox/stories/index.stories.tsx

```diff
@@ -1,68 +1,40 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable unicorn/consistent-destructuring */
 import * as React from "react";
-import { InputBox } from "../../input-box";
-import { Box } from "../../../primitives/box";
-import {
-  StyledComboboxItem,
-  StyledCombobox,
-  useComboboxState,
-} from "../../combobox";
-import {
-  SelectComboboxItem,
-  useSelectComboboxState,
-  SelectCombobox,
-  SelectComboboxPopover,
-  SelectComboboxList,
-} from "../index";
+import { useUID } from "react-uid";
+import { useComboboxState } from "../../combobox";
+import { SelectCombobox, useSelectComboboxState } from "../index";
 import { itemList as list } from "../../combobox/__fixtures__/item-list";
 
 // eslint-disable-next-line import/no-default-export
 export default {
   title: "Components/SelectCombobox",
+  component: SelectCombobox,
 };
 
 export const Default: React.FC = () => {
   const combobox = useComboboxState({ list, gutter: 4, sameWidth: true });
-  // value and setValue shouldn't be passed to the select state because the
-  // select value and the combobox value are different things.
-  // value and setValue are in the combobox state
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { value, setValue, ...selectProps } = combobox;
   const select = useSelectComboboxState({
     ...selectProps,
     defaultValue: "Apple",
   });
 
-  // Resets combobox value when popover is collapsed
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // eslint-disable-next-line unicorn/consistent-destructuring
   if (!select.mounted && combobox.value) {
     combobox.setValue("");
   }
```

Comment:
```suggestion
  if (!select.mounted && value) {
    setValue("");
  }
```
If you use the destructured value and setValue you don't have to disable any linter errors.